securitas/tests/unit/controller/test_password.py

```diff
@@ -9,3 +10,105 @@ def test_password_reset(client):
+
+
+@pytest.mark.vcr()
+def test_password(client):
+    """Verify that current password must be correct"""
+    result = client.post(
+        '/password-reset',
+        data={
+            "username": "admin",
+            "current_password": "1",
+            "password": "LongSuperSafePassword",
+            "password_confirm": "LongSuperSafePassword",
+        },
+        follow_redirects=True,
+    )
+    assert result.status_code == 200
+    page = BeautifulSoup(result.data, 'html.parser')
+    password_input = page.select("input[name='current_password']")[0]
+    assert 'invalid' in password_input['class']
+    helper_text = password_input.find_next("span", class_="helper-text")
+    assert helper_text["data-error"] == "The old password or username is not correct"
+
+
+@pytest.mark.vcr()
+def test_time_sensitive_password_policy(client, dummy_user):
+    """Verify that new password policies are upheld"""
+    result = client.post(
+        '/password-reset',
+        data={
+            "username": "dummy",
+            "current_password": "dummy_password",
+            "password": "somesupersecretpassword",
+            "password_confirm": "somesupersecretpassword",
+        },
+        follow_redirects=True,
+    )
+    page = BeautifulSoup(result.data, 'html.parser')
+    password_input = page.select("input[name='password']")[0]
+    assert 'invalid' in password_input['class']
+    helper_text = password_input.find_next("span", class_="helper-text")
+    assert helper_text["data-error"] == "Constraint violation: Too soon to change password"
+
+
+@pytest.mark.vcr()
+def test_short_password(client, no_password_min_time):
+    """Verify that new password policies are upheld"""
+    result = client.post(
+        '/password-reset',
+        data={
+            "username": "dummy",
+            "current_password": "dummy_password",
+            "password": "1",
+            "password_confirm": "1",
+        },
+        follow_redirects=True,
+    )
+    page = BeautifulSoup(result.data, 'html.parser')
+    password_input = page.select("input[name='password']")[0]
+    assert 'invalid' in password_input['class']
+    helper_text = password_input.find_next("span", class_="helper-text")
+    assert helper_text["data-error"] == "Constraint violation: Password is too short"
+
+
+@pytest.mark.vcr()
+def test_password_changes(client, no_password_min_time):
+    """Verify that password changes"""
+    result = client.post(
+        '/password-reset',
+        data={
+            "username": "dummy",
+            "current_password": "dummy_password",
+            "password": "secretpw",
+            "password_confirm": "secretpw",
+        },
+        follow_redirects=True,
+    )
+    assert result.status_code == 200
+    page = BeautifulSoup(result.data, 'html.parser')
+    messages = page.select(".flash-messages .green")
+    assert len(messages) == 1
+    assert messages[0].get_text(strip=True) == \
```

Comment:
In Python it's more common to use the parenthesis than the backslash to handle line breaks, so you'd have something like:
```python
assert messages[0].get_text(strip=True) == (
    'Your password has been changed, please try to log in with the new one now.'
)
```
But then again, if Black is happy with it I have nothing to say ;-)

Reply:
Ran black again, made some changes. Thanks!